Solved "findComponentById() return null for custom component" issue.

Click2cloud-Eros opened the pull request:
**What type of PR is this?**
bug fix

**What this PR does / why we need it?**
This PR included some major and minor bugs fixes.

**Which issue(s) this PR fixes:**
Fixes #1 
Fixes # add correct equivalent of onMesure and onSizeChanged method and commented manually created method.
Fixes #3 
Fixes # double-tap zoom feature is not working. 
Fixes # long click event is not working. 
Fixes # react is failed to draw, because of wrong parameters. 
Fixes # Solved some other crash issues.

Click2cloud-Eros committed:
1) Solved "findComponentById() return null for custom component" issue.
2) add correct equivalent of onMesure and onSizeChanged method and commented manually created method.
3) Solved some other crash issues.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/applibgroup/Pan_and_Zoom?pullRequest=2) <br/>All committers have signed the CLA.

sharathks84 commented on hdimageview/src/main/java/xyz/zpayh/hdimageview/datasource/interceptor/NetworkInterceptor.java:
```diff
@@ -102,16 +102,13 @@ public ImageSource intercept(Chain chain) throws IOException, NotExistException,
-                //e.printStackTrace();
-                return Interceptors.fixJPEGDecoder(file,e);
-            }
+            /*                //InputStream inputStream = processBitmap(uri.toString());
```
remove the comments from the code

sharathks84 commented on hdimageview/src/main/java/xyz/zpayh/hdimageview/core/HDImageViewConfig.java:
```diff
@@ -24,17 +24,8 @@
 //import androidx.annotation.Nullable;
 //import android.view.animation.DecelerateInterpolator;
 //import android.view.animation.Interpolator;
```
remove commented code in all the codebase

sharathks84 commented on hdimageview/src/main/java/xyz/zpayh/hdimageview/SimpleValueAnimator.java:
```diff
@@ -141,7 +141,10 @@ public void run() {
             } else {
 //                HandlerActionQueue handlerActionQueue = new HandlerActionQueue();
```
every file has comments which should be removed. Too many unwanted code

Click2cloud-Eros replied:
Yes, Actually this library is not completely implemented still some issues are not resolve.

Click2cloud-Eros committed:
1) Fixed an issue where "Attempt to update UI in non-UI thread" crash happen.
2) Fixed an issue where double-tap zoom feature is not working.
3) Fixed an issue where long click event is not working.
4) Fixed an issue where react is failed to draw because of wrong parameters.
5) Do some changes for solving drag functionality but not able to solve it.
